week03/assignment03.go

```diff
@@ -0,0 +1,107 @@
+package week03
+
+import (
+	"fmt"
+)
+
+var (
+	ErrRatingNoPlays       = fmt.Errorf("can't review a movie without watching it first")
+	ErrInvalidViewersValue = fmt.Errorf("invalid number of viewers, must be greather than 0")
+	ErrTeatherPlayNoMovies = fmt.Errorf("no movies to play")
+)
+
+type (
```

Comment:
While you can do this in Go with `type`, please don't. It is idiomatic to use this pattern with `var` and `const`, but not with `type`.

Reply:
Thanks Mark! I saw it on the temporal codebase an thought "Why not?" thanks for clarifying it.